apps/remote_control/lib/lexical/remote_control/search/indexer/entry.ex

```diff
@@ -0,0 +1,90 @@
+defmodule Lexical.RemoteControl.Search.Indexer.Entry do
+  @type entry_type :: :module | :function | :macro | :field
```

Comment:
> I'm not sure that :function and :macro should be separate, since they exist in the same namespace.

What's the harm of being specific here. For find references, this is right, but I think it might be helpful to differentiate between the parts of speech of elixir. Calling something a function a macro seems wrong to me. If you want both, we should make it possible to query for both, possibly by passing a list into the query function.

Reply:
for the `entry_type`,  maybe I should just remove everything except `:module`